### ISSUE TITLE
Issue with "UI Tour": Images Missing (301 moved permanently)

### ISSUE THREAD

**Issue by mplanchard** [mdn/content#963]
MDN URL: https://developer.mozilla.org/en-US/docs/Tools/Debugger/UI_Tour

#### What information was incorrect, unhelpful, or incomplete?

Looks like the images on this page are all returning 301s, for example https://mdn.mozillademos.org/files/16841/debugger_uiTour_02.png

Example response headers:

```
HTTP/2 301 Moved Permanently
content-type: text/html; charset=utf-8
content-length: 0
location: https://mdn.mozillademos.org/files/16841/debugger_uiTour_02.png
cache-control: public, max-age=900
content-language: en-US
date: Tue, 05 Jan 2021 15:31:43 GMT
etag: "d41d8cd98f00b204e9800998ecf8427e"
server: gunicorn/20.0.4
strict-transport-security: max-age=63072000
x-content-type-options: nosniff
x-frame-options: DENY
x-xss-protection: 1; mode=block
x-cache: Hit from cloudfront
via: 1.1 f47ebd7d279bfcb757b1e0e59e2a3cd4.cloudfront.net (CloudFront)
x-amz-cf-pop: DFW55-C2
x-amz-cf-id: 1apEZ4MivRiI_PBfiKS7Zy_sdapdd5Dfe0bG7ud6sVYXe2WOnB5uNA==
age: 102
X-Firefox-Spdy: h2
```

#### Specific section or headline?

All of them

#### What did you expect to see?

Images

#### Did you test this? If so, how?

Loaded with and without ad blockers.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/tools/debugger/ui_tour`
* MDN URL: https://developer.mozilla.org/en-US/docs/Tools/Debugger/UI_Tour
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/tools/debugger/ui_tour/index.html
* Report started: 2021-01-05T15:32:03.422Z

</details>

**Comment by M-Miller-42** [comment]
This seems to be the case for ALL images on MDM sites. 
Other examples:

-  [Planning a simple website](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure#Planning_a_simple_website)
- [Type-specific rich output](https://developer.mozilla.org/en-US/docs/Tools/Web_Console/Rich_output#Type-specific_rich_output)
-   [Service workers demo](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers#Service_workers_demo)

**Comment by hamishwillee** [comment]
@mplanchard @M-Miller-42 et al. This is a known issue. 

We're moving all the images to be locally stored locally in this repo.  If you build the source locally there is an option to "Fix flaws" displayed on the page. If you click this it automatically fetches the images for you and puts them into the source tree. As an example, I have fixed this issue in https://github.com/mdn/content/pull/1600.

Closing this issue because "we are on it". Thanks very much for posting though.
